sections/semantics-embedded-content.include

```diff
@@ -7544,7 +7547,7 @@ zero or more <{track}> elements, then
   The <dfn attribute for="HTMLMediaElement"><code>currentTime</code></dfn> attribute must, on
   getting, return the <a>media element</a>'s <a>default playback start position</a>,
   unless that is zero, in which case it must return the element's <a>official playback
-  position</a>. The returned value must be expressed in seconds. If the
+  position</a>. The returned value must be expressed in seconds. On setting, if the
   <a href="#media-elements">media element</a>'s <code>readyState</code> is <code>HAVE_NOTHING</code>,
```

Comment:
Change:
`<a href="#media-elements">media element</a>`
to
`<a>media element</a>`